src/shared.ts

```diff
@@ -19,6 +19,8 @@ type Shared = {
     options: CodeMirror.TextMarkerOptions;
   }>;
   announcer: HTMLElement;
+  keyMap: {[index:string]: unknown};
+  keyName: (event: React.KeyboardEvent) => string;
```

Comment:
As far as I could tell, these never actually get set, but they are referenced by PrimitiveList. Possibly a bug?

Reply:
DEFINITELY a bug. I used to dump these dictionaries in the shared store, because they were used in multiple places. Once I consolidated them into `keymap.js^H^Htsx`, I forgot to remove this reference _and_ to export those constants from the keymap file. I'll push a fix to master momentarily.

Reply:
This is now fixed on master, so you can safely remove these lines.

Reply:
excellent! I like not adding things to SHARED :smiley: